README.md

```diff
@@ -1,23 +1,45 @@
 ISO 15118 library suite
 =======================
 
-Currently, this is a proof of concept library.
+This is a C++ library implementation of ISO 15118 for part 20.
 
-Integration of ISO 15118 development for EVerest is under heavy development
-currently. Many features are already implemented, many are still to follow.
+Integration of ISO15118-20 development for EVerest is under heavy development
+currently. 
+
+## ISO 15118-20 Support
+
+The following tables show the current support for the listed EVSE ISO15118-20 features.
+
+| Feature                            | Supported          |
+|------------------------------------|--------------------|
+| TCP, TLS 1.2 & 1.3                 | :heavy_check_mark: |
+| DC, DC_BPT                         | :heavy_check_mark: |
+| AC, AC_BPT                         | WIP                |
+| WPT                                |                    |
+| ACDP                               |                    |
+| ExternalPayment                    | :heavy_check_mark: |
+| Plug&Charge                        | WIP                |
+| CertificateInstallation            |                    |
+| Scheduled                          | :heavy_check_mark: |
+| Dynamic Mode (+ MobilityNeedsMode) | :heavy_check_mark: |
+| Private Env                        |                    |
+| Pause/Resume & Standby             | WIP                |
+| Schedule renegotation              |                    |
+| Smart Charging                     |                    |
+| Multiplex messages                 |                    |
+| Internet service                   |                    |
+| Parking status service             |                    |
 
-But: This repository here is not the place where implementation happens.
```

Comment:
I would use something like

`ISO15118 support is distributed accross multiple repositories and modules in EVerest. Please see the following references of other ISO15118 related development:`

Reply:
You can then also drop this last sentence:

`This is just a short overview to show...`

Reply:
:+1: